src/olympia/devhub/templates/devhub/addons/submit/source.html

```diff
@@ -64,7 +64,7 @@ <h3>{{ _("You must submit your extension's source code.") }}</h3>
             {% endtrans %}
         </p>
         <p>
-            {% trans a_attrs='href="%s/documentation/publish/source-code-submission?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=submission" target="_blank" rel="noopener noreferrer"'|format(settings.EXTENSION_WORKSHOP_URL)|safe %}
+            {% trans a_attrs='href="%s/documentation/publish/source-code-submission/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=submission" target="_blank" rel="noopener noreferrer"'|format(settings.EXTENSION_WORKSHOP_URL)|safe %}
```

Comment:
lets make the utm parameters consistent while we're making changes.  Can you change them to `utm_medium=devhub&utm_content=submission-flow`